self-hosting/prepare-system/ubuntu/supervisor.conf

```diff
@@ -1,7 +1,7 @@
 
 [program: medic]
 process_name=%(program_name)s
-command=sudo /usr/local/bin/docker-compose -f /home/medic/self-hosting/main/docker-compose.yml up
+command=sudo /usr/local/bin/docker-compose -f /home/medic/self-hosting/main/docker-compose.yml up -d
```

Comment:
Does this not cause supervisord to think that the process that was started has crashed and try to restart it over and over again? (i.e. because the process that's launched exits right away if you use `-d`).

Reply:
I have redirected output to the null device instead since the docker output for all the containers is already sent to the logs anyway.

Reply:
Cool, yeah I think that's the better approach. LGTM.